Header Header

int32 red_beacon
int32 blue_beacon
int32 green_beacon